src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
```

Comment:
For the future -> better not to use such short names. You can name it `classnames` here